libs/utils/android/workload.py

```diff
@@ -62,7 +62,7 @@ def _check_availables(cls, test_env):
         _log.debug('Building list of available workloads...')
         for sc in Workload.__subclasses__():
             _log.debug('Checking workload [%s]...', sc.__name__)
-            if sc.package in cls._packages:
+            if sc.package in cls._packages or sc.package == '':
```

Comment:
I had a talk with @derkling about that. Empty string was suggested so that it is obvious to the developer that his workload requires no package (arguably that could also apply to None)